semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -83,9 +83,11 @@
           attributeValueMap.get(HTTP_HOST.getValue()).getValue(),
           attributeValueMap.get(HTTP_TARGET.getValue()).getValue());
         return Optional.of(url);
-    } else if (SpanSemanticConventionUtils.isClientSpanForOtelFormat(attributeValueMap)) {
+    } else if (SpanSemanticConventionUtils.isClientSpanForOtelFormat(attributeValueMap)
```

Comment:
Just a note: we want to treat both `span_kind` and `span.kind`, can we just extend `isClientSpanForOtelFormat`, so that changes will be more readable, we can add a comment there too and if requires can be used multiple places?

Reply:
For now keeping it separate seems a clean way.